### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Bouh**

### CODE REVIEW

File: newIDE/app/src/CommandPalette/CommandManager.js [hunk 1]
```diff
@@ -67,8 +67,8 @@ export default class CommandManager implements CommandManagerInterface {
 
   getAllNamedCommands = () => {
     return Object.keys(this._commands).map<NamedCommand>(commandName => {
-      const cmd = this._commands[commandName];
-      return { ...cmd, name: commandName };
+      const command = this._commands[commandName];
+      return { ...(command: Command), name: commandName };
```
Comment (Bouh) [inline newIDE/app/src/CommandPalette/CommandManager.js:71]: Same patern as the lines above, with the type.

File: newIDE/app/src/ObjectEditor/Editors/EmptyEditor.js [hunk 2]
```diff
@@ -1,24 +1,20 @@
 // @flow
 import { Trans } from '@lingui/macro';
-
 import * as React from 'react';
-import { type EditorProps } from './EditorProps.flow';
 import { Line, Column } from '../../UI/Grid';
 import EmptyMessage from '../../UI/EmptyMessage';
 
-export default class EmptyEditor extends React.Component<EditorProps, void> {
```
Comment (Bouh) [inline newIDE/app/src/ObjectEditor/Editors/EmptyEditor.js:9]: Convert into functionnal component, and remove useless type.

**Comment by Bouh** [comment]
Ready to merge

**Review by 4ian — APPROVED** [review]
Thank you!

**Pull request merged**